aws_lambda_powertools/logging/formatter.py

```diff
@@ -49,9 +49,7 @@ def _build_root_keys(**kwargs):
     @staticmethod
     def _get_latest_trace_id():
         xray_trace_id = os.getenv("_X_AMZN_TRACE_ID")
-        trace_id = xray_trace_id.split(";")[0].replace("Root=", "") if xray_trace_id else None
```

Comment:
could you revert this? got caught once trying to debug examples like this, and that's the only reason it has a dedicated assignment despite looking useless at first glance

Reply:
nvm, misread the diff - I thought `getenv` was being touched altogether.